README.md

```diff
@@ -1,66 +1,128 @@
+   for the xCORE VocalFusion XVF3100.
 
-9. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+   Read and accept the AVS Device SDK license agreement.
+
+8. You will be asked whether you want the Sample App to run automatically when the Raspberry Pi boots. It is recommended that you respond "yes" to this option.
+
+9. Read and accept the Sensory license agreement. Wait for the script to complete the installation. The script is configuring the Raspberry Pi audio system, downloading and updating dependencies, building and configuring the AVS Device SDK. It takes around 30 minutes to complete.
+
+10. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+
+11. If you selected the option to run the Sample App on boot you should now be able to complete the registration by following the steps from 2 onward here:
+https://github.com/alexa/avs-device-sdk/wiki/Raspberry-Pi-Quick-Start-Guide-with-Script#finish-authorization-using-login-with-amazon
+
+12. Now you can execute an AVS command such as "Alexa, what time is it?".
+
+   On the XMOS **xCORE VocalFusion XVF3510 Kit for Amazon AVS**, the LED on the Pi HAT board will change colour when the system hears the "Alexa" keyword, and will then cycle back and forth whilst waiting for a response from the Amazon AVS server.
+
+   On the XMOS **xCORE VocalFusion Stereo 4-Mic Kit for Amazon AVS**, the LEDs on the development board should reflect the approximate direction from which the microphones are receiving a stimulus.
 
 ## Running the AVS SDK Sample App
-The following commands will be available to run from a terminal:
+The automated installation script creates a number of aliases which can be used to execute the AVS Device SDK client, or run the unit tests:
 - `avsrun` to run the Sample App.
-- `avsauth` to re-authenticate your Alexa device details.
-- `avsunit` to run the unit tests.
-- `avssetup` to re-install the Sample App.
+- `avsunit` to run the unit tests (invoke Amazon's `test.sh`).
+- `avssetup` to re-install the Sample App (re-run XMOS modified `setup.sh`).
+
+## Using different Amazon details
+To change client and product ID, download your updated config.json and in the same folder run `avssetup config.json rebuild`. Subsequent `avsrun` calls will use the new details.
+
+## Changing Sensory operating point
+
+To change to operating point of the Sensory keyword engine, navigate to 
+```
+~/sdk-folder/avs-device-sdk/KWD/Sensory/src/
+```
+   
+Change line 61 in `SensoryKeywordDetector.cpp` to your desired value:
+
+```c++
+/// The Sensory operating point
+static const unsigned int SENSORY_OPERATING_POINT = 12;
+```
+
```

Comment:
This operation it be done now directly by changing the avsrun alias, it is set to 12 by default.

Reply:
I have changed this to describe which file and how to edit, lmk if it works for you...